src/rescoring.js

```diff
@@ -1561,7 +1562,15 @@ const RescoringContentTableRow = ({
               sx={{ marginY: '0.5rem' }}
             >
               {
-                findingCfg.categorisations.map((categorisation) => <MenuItem
+                findingCfg.categorisations.filter((categorisation) => {
+                  if (!categorisation.rescoring) return false
+
+                  if (Array.isArray(categorisation.rescoring)) {
+                    return categorisation.rescoring.includes(RESCORING_MODES.MANUAL)
```

Comment:
hmm... would it make sense to normalise to array/list on backend-side?
or asked differently: why do we have both flavours?